README.md

```diff
@@ -6,6 +6,78 @@
 ![Screenshots with my stupid grinning face](images/screenshot.jpg)
 (Credits for the nice backgrounds to [Mary Sabell](https://dribbble.com/shots/4686178-Bauhaus-Poster) and [PhotoFunia](https://photofunia.com/effects/retro-wave))
 
+## Maintainers
+
+  * Phil Ashby (@phlash)
+  * Benny Baumann (@benbe)
+  * Florian Echtler (@floe)
+
+## Building
+
+Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
+
+Clone this repository with `git clone --recursive https://github.com/floe/backscrub.git`.
+To speed up the checkout you can additionally pass `--depth=1` to `git clone`.
+This is okay, if you only want to download and build the code, however, for development it is not recommended.
+
+Use `cmake` to build the project: create a subfolder (e.g. `build`), change to that folder and run: `cmake && make -j4` (please replace "4" with the number of your cores).
+
+Alternatively, you can also run `make` to build everything (should also clone and build Tensorflow Lite).
+
+## Usage
+
+First, load the v4l2loopback module (extra settings needed to make Chrome work):
+```
+sudo modprobe v4l2loopback devices=1 max_buffers=2 exclusive_caps=1 card_label="VirtualCam"
+```
+Then, run deepseg (-d -d for full debug, -c for capture device, -v for virtual device):
+```
+./deepseg -d -d -c /dev/video0 -v /dev/video1
+```
+
+Some cameras like a *Logitec Brio* might need to switch the video source to `MJPG` by passing `-f MJPG` in order to allow for higher resolutions.
+
+## Requirements
+
+Tested with the following dependencies:
+
+  - Ubuntu 20.04, x86-64
+    - Linux kernel 5.6 (stock package)
+    - OpenCV 4.2.0 (stock package)
+    - V4L2-Loopback 0.12.5 (stock package)
+    - Tensorflow Lite 2.5.0 (from [repo](https://github.com/tensorflow/tensorflow/tree/v2.5.0/tensorflow/lite))
+  - Ubuntu 18.04.5, x86-64
+    - Linux kernel 4.15 (stock package)
+    - OpenCV 3.2.0 (stock package)
+    - V4L2-Loopback 0.10.0 (stock package)
+    - Tensorflow Lite 2.1.0 (from [repo](https://github.com/tensorflow/tensorflow/tree/v2.1.0/tensorflow/lite))
+  
```

Comment:
Can also add my Debian 10 (stable) with stock kernel, opencv, v4l2loopback and TFlite 2.5.0

Reply:
Please do :-)